tfmkt/spiders/players.py

```diff
@@ -87,6 +89,20 @@ def parse_details(self, response, base):
           href
         )
 
+    # parse historical market value
+    pattern = re.compile('\'data\'\:.*\}\}]')
+
+    d = (
+        json
+        .loads('{' + response.xpath("//script[contains(., 'series')]/text()").re(pattern)[0].replace("\'", "\"").encode().decode('unicode_escape') + '}')
+    )
+
+    for x in d['data']:
```

Comment:
Is there a reason for the extra data level within the market_value_history field?
I mean, why not just
```json
{
  "market_value_history": [
    {
        "verein": "Utsiktens BK",
        "age": "19",
        "mw": "-",
        "datum_mw": "Apr 5, 2018",
        "marker": {
          "symbol": "url(https://tmssl.akamaized.net/images/wappen/tiny/13653.png?lm=1461965079)"
        },
        "date_x": 1522879200000,
        "market_value": 0
      }
  ]
}
```
instead of
```json
{
  "market_value_history": {
    "data": [
      {
         "verein": "Utsiktens BK",
         "age": "19",
         "mw": "-",
         "datum_mw": "Apr 5, 2018",
         "marker": {
          "symbol": "url(https://tmssl.akamaized.net/images/wappen/tiny/13653.png?lm=1461965079)"
         },
         "date_x": 1522879200000,
         "market_value": 0
       }
    ]
  }
}

```